### ISSUE TITLE
AuthLink doesn't accept any parameter in version 1.0.0

### ISSUE THREAD

**Issue by bCliks** [zino-app/graphql-flutter#227]
**Describe the bug**
I am using version `graphql_flutter: ^1.0.0`
And trying to assign Bearer access token in AuthLink class. But it doesn't accept any parameter.

**Screenshots**
![image](https://user-images.githubusercontent.com/3747809/56490957-92485800-6504-11e9-99cf-1b91109c6717.png)

[I am following this guide to install ](https://pub.dartlang.org/packages/graphql_flutter#installation)

**Comment by eusdima** [comment]
I am trying to reproduce what you are getting but my AuthLink seems fine.

Can you hover over AuthLink and see what it shows? This is what is supposed to be displaying.
<img width="445" alt="Screenshot 2019-04-22 at 10 28 06" src="https://user-images.githubusercontent.com/4757453/56491597-666ba900-64e9-11e9-8f86-bef3b09141fc.png">

What version of Dart/Flutter do you have?

**Comment by mainawycliffe** [comment]
@bCliks  Can you try it like this:

```
final AuthLink authLink = AuthLink(
      getToken: () async => 'XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX',
    );
```

**Comment by chenxianqi** [comment]
It seems that the same problem is here.
![image](https://user-images.githubusercontent.com/27764334/56501753-7c598800-6542-11e9-8984-1a7c8e73bead.png)

**Comment by mainawycliffe** [comment]
You need to cast `httpLink as Link` for that to work

```dart
final Link link = authLink.concat(httpLink as Link);
```

**Comment by chenxianqi** [comment]
> ```dart
> httpLink as Link
> ```

Perfect jie'j Thank

**Comment by micimize** [comment]
resolved by #229
